### ISSUE TITLE
Overblog GraphQL ObjectTypeExtensionNode definition is not supported right now

### ISSUE THREAD

**Issue by nkamuo** [overblog/GraphQLBundle#1032]
| Q                | A
| ---------------- | -----
| Bug report?      | yes/no
| Feature request? | yes
| BC Break report? | yes/no
| RFC?             | yes/no
| Version/Branch   | 0.14.3

I am unable to use the graphql specification syntax;
extend type TypeName{
filedName: Type
filedName2: Type
}
is this a bug or is there a way to resolve this soon enough for my project that might require the feature.
Thank you.

**Comment by MGDSoft** [comment]
Yep, I also want to split my code into different files. @mcg-web what do you think about this? I'm open to trying to solve it but may need some guidance. Any recommendations?
